File: Coding/.vscode/C++_Python/baekjoon_2941.py
#https://www.acmicpc.net/problem/2941 - 크로아티아 알파벳
croatia = ["c=","c-","dz=","d-","lj","nj","s=","z="]
alphabet = input()[:100]

swich = 0
for char in croatia:
    swich = alphabet.find(char)
    if(swich != -1):
        alphabet = alphabet.replace(char,"1")
print(len(alphabet))

File: Coding/.vscode/C++_Python/Step_7_String/baekjoon_2941.cpp
//https://www.acmicpc.net/problem/2941 - 크로아티아 알파벳
#include<bits/stdc++.h>
using namespace std;

int main(){
    string croatia[8] = {"c=", "c-", "dz=", "d-", "lj", "nj", "s=", "z="};
    string T;
    cin >> T;
    
    for (int i = 0; i < 8; i++)
    {
        int s = 0;
        s = T.find(croatia[i]);
        if(s != -1){
            T.replace(T.find(croatia[i]),croatia[i].size(),"1");
            i--;
        }
    }
    cout << T.length() << "\n";
}